src/app.js

```diff
@@ -579,15 +576,15 @@ fluid.defaults("gpii.app.menu", {
             singleTransform: {
                 type: "fluid.transforms.free",
                 func: "gpii.app.menu.generateMenuTemplate",
-                args: ["{that}.model.keyedInUser", "{that}.model.keyOut", "{that}.options.snapsets", "{that}.options.exit"]
+                args: ["{that}.model.keyedInUser", "{that}.options.snapsets", "{that}.model.keyOut", "{that}.options.exit"]
             },
             priority: "last"
         }
     },
     menuLabels: {
         keyedIn: "Keyed in with %userTokenName",    // string template
```

Comment:
The wireframes at GPII-2527 suggest we should not display the token name. This is quite reasonable given that in most cases we plan to deploy it will be an impenetrable GUID. They suggest instead displaying the preference set name, assuming that the preference set includes more than one